apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -190,6 +192,20 @@ GoogleBlockly.Extensions.registerMutator(
   procedureDefMutator
 );
 
+// TODO: After updating to Blockly v10, use the original
+// procedure_caller_mutator and procedure_caller_on_change_mixin.
```

Comment:
Mind linking? https://codedotorg.atlassian.net/browse/CT-148

Maybe overkill, but there's a lot to remember to do after we upgrade 😄